.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
 data/
+package.json
```

Comment:
```suggestion

```
package.json is quite important to install the packages to run the project. I would not add this to .gitignore

Reply:
Ah okay. I'm not very experienced in using Node.js and Typescript, so I was unaware of this. It suddenly got added on my end as an untracked file, so I thought that it wasn't supposed to be present in the repo.

Reply:
Yeah, I know which one you are talking about. 
This one gets installed due to the install.sh script. 
Maybe we should just add that generated package.json to github.